### ISSUE TITLE
How to render the bar magnet?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/faradays-electromagnetic-lab#21]
How should the bar magnet be rendered in the HTML5 version of this sim?

If the Java version (and early HTML5 versions), the bar magnet is a PNG image file, 

![barMagnet](https://github.com/phetsims/faradays-electromagnetic-lab/assets/3046552/a1ed4e81-bbeb-43b1-8223-f2ebbe2d1ba2)

There are several problems with this:

1. 'N' and 'S' labels are not translatable
2. Colors cannot be changed via a color profile
3. The lighting does not update when the bar magnet is rotated.

If we do not use an image file, we will lose the following characteristics:

1. The texture on the magnet
2. The embossing of the 'N' and 'S' labels

**Comment by pixelzoom** [comment]
12/4/2023 design meeeting: @arouinfar @ariel-phet @kathy-phet @brent-phet @pixelzoom

Draw the bar magnet in code.  OK to lose texture and embossing.

**Comment by pixelzoom** [comment]
Drawing the bar magnet is code was completed awhile ago, see screenshot below.  Consensus is that this looks good, and the new colors are a good tie-in with Faraday's Law.   We also decided not change the lighting or position of the "N" and "S" labels when the magnet is rotated. 

Closing.

<img width="302" alt="screenshot_2992" src="https://github.com/phetsims/faradays-electromagnetic-lab/assets/3046552/9fee4d6f-21e6-4fd8-a133-81bf1097fc04">
